sidecar/config/config.go

```diff
@@ -426,3 +450,66 @@ func parseServiceNameAndTags(service string) (name string, tags []string) {
 	}
 	return
 }
+
+type A8ServicesNginxConfData struct {
```

Comment:
Let's avoid prefixing structs/consts/variables with `A8` anywhere since this is the A8 code base. A8 is implicit.

This comment applies to the entire commit.